mino/minogrpc/controller/mod.go

```diff
@@ -1,26 +1,43 @@
 package controller
 
 import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"io"
 	"math"
+	"path/filepath"
 	"time"
 
 	"go.dedis.ch/dela"
 	"go.dedis.ch/dela/cli"
 	"go.dedis.ch/dela/cli/node"
+	"go.dedis.ch/dela/core/store/kv"
+	"go.dedis.ch/dela/crypto/loader"
 	"go.dedis.ch/dela/mino"
 	"go.dedis.ch/dela/mino/minogrpc"
+	"go.dedis.ch/dela/mino/minogrpc/certs"
 	"go.dedis.ch/dela/mino/router/tree"
 	"golang.org/x/xerrors"
 )
 
+const certKeyName = "cert.key"
```

Comment:
is there any need for this to be configurable?
I'm not sure when this implementation would be used.

Reply:
More than to be configurable, it's simply good practice to use constants when the code is using a value like this. That's what I learnt, but there's probably different mindsets.

Reply:
Oh, I agree with you entirely.
I was just asking if this was a hardcoded constant or if later on it might need to be configured.

Reply:
Then the answer is yes, that's only a hardcoded constant, which will probably evolve to a default name and an option to set a different name.